Samples/Xbox/InteractivitySample.cpp

```diff
@@ -32,6 +33,7 @@ namespace
     const int c_joystickXByParticipantReadingLabel          = 1019;
     const int c_joystickYByParticipantReadingLabel          = 1020;
 
+    // TODO: Replace the Project Version ID with your Project Version ID from Interactive Studio

```

Comment:
Oh! that makes sense.

---
In reply to: [132344028](https://github.com/mixer/interactive-cpp/pull/25#discussion_r132344028) [](ancestors = 132344028,132342571)